hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -76,8 +76,10 @@ public class HiveSyncConfig extends HoodieSyncConfig {
   @Parameter(names = {"--help", "-h"}, help = true)
   public Boolean help = false;
 
-  @Parameter(names = {"--support-timestamp"}, description = "'INT64' with original type TIMESTAMP_MICROS is converted to hive 'timestamp' type."
-      + "Disabled by default for backward compatibility.")
+  @Parameter(names = {"--support-timestamp"},
```

Comment:
Do we need similar config in Presto/Trino as well?

Reply:
@cdmikechen @XuQianJin-Stars can you follow up on this comment pls? what would be the plan for presto/trino?